sources/install/package_misc.sh

```diff
@@ -208,6 +208,25 @@ function install_wesng() {
     add-to-list "wesng,https://github.com/bitsadmin/wesng,WES-NG is a tool based on the output of Windows's systeminfo utility which provides the list of vulnerabilities the OS is vulnerable to including any exploits for these vulnerabilities."
 }
 
+function install_glow() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history
+    colorecho "Installing glow"
+    wget https://github.com/charmbracelet/glow/releases/download/v2.1.0/glow_2.1.0_Linux_x86_64.tar.gz -O /tmp/glow.tar.gz
+    tar -xvf /tmp/glow.tar.gz
+    cp glow_2.1.0_Linux_x86_64/glow /opt/tools/bin
+    rm -f /tmp/glow.tar.gz
+    rm -rf /tmp/glow_2.1.0_Linux_x86_64/
+    add-test-command "glow --help"
+    add-to-list "glow,https://github.com/charmbracelet/glow,glow is a tool to render Markdown inside the terminal."
+}
+
+function install_thr() {
+    # CODE-CHECK-WHITELIST=add-aliases,add-history
+    colorecho "Installing thr"
+    git -C /opt/tools/ clone https://github.com/The-Hacker-Recipes/The-Hacker-Recipes.git
+    add-test-command "ls /opt/tools/The-Hacker-Recipes/"
+    add-to-list "thr,https://www.thehacker.recipes/,THR (The Hacker Recipes) is aimed at providing technical guides on various hacking topics."
```

Comment:
we should probably include the "server install" in here, or an alias, that allows to actually run THR locally 🤷